### ISSUE TITLE
Changing the DOM after the popper was created

### ISSUE THREAD

**Issue by syzhakov** [FezVrasta/popper.js#741]
### Steps to reproduce the problem

1. create a popper instance (new Popper(...))
2.  add an element to the DOM, above the referenced element (I'm using Vue but jQuery would work as well)
3.  the popper will not point to the referenced element anymore, but will stay at the old position

### What is the expected behavior?

When the DOM changes and the referenced element moves, the popper will follow it

**Comment by syzhakov** [comment]
I agree about the expensiveness issue, but I would expect it to follow the DOM behaviour just as any other element. Isn't there a way to change the origin of the translate3d property? If it would follow the origin I choose I can ensure that it is kept aligned to the other content..

**Comment by FezVrasta** [comment]
If you want to make it relative to some given parent, you will have to put the popper inside that container and set `position: relative` and `overflow: hidden` to the container itself.

**Comment by FezVrasta** [comment]
I think the container must also be `overflow: hidden` to work properly, but in my opinion it would make more sense to trigger the `scheduleUpdate` when you need it.
